### ISSUE TITLE
Default presentation scheme to 'file://'

### ISSUE THREAD

**Issue by sinedied** [astefanutti/decktape#108]
With the old `1.0.0` version I was able to convert local file files (ie from the filesystem).

I tested the new v2 version, and I have have issue while trying to do the same thing:
```
decktape -p 1000 .tmp/pdf/index.html pdf/index.pdf
Error: Protocol error (Page.navigate): Cannot navigate to invalid URL undefined
    at Session._onMessage (/Users/noda/github/backslide/node_modules/puppeteer/lib/Connection.js:194:25)
    at Connection._onMessage (/Users/noda/github/backslide/node_modules/puppeteer/lib/Connection.js:98:19)
    at emitOne (events.js:96:13)
    at WebSocket.emit (events.js:191:7)
    at Receiver._receiver.onmessage (/Users/noda/github/backslide/node_modules/puppeteer/node_modules/ws/lib/WebSocket.js:143:47)
    at Receiver.dataMessage (/Users/noda/github/backslide/node_modules/puppeteer/node_modules/ws/lib/Receiver.js:389:14)
    at Receiver.getData (/Users/noda/github/backslide/node_modules/puppeteer/node_modules/ws/lib/Receiver.js:330:12)
    at Receiver.startLoop (/Users/noda/github/backslide/node_modules/puppeteer/node_modules/ws/lib/Receiver.js:165:16)
    at Receiver.add (/Users/noda/github/backslide/node_modules/puppeteer/node_modules/ws/lib/Receiver.js:139:10)
    at Socket._ultron.on (/Users/noda/github/backslide/node_modules/puppeteer/node_modules/ws/lib/WebSocket.js:139:22)
```
As a workaround, I added `file://` + the absolute path to make it work.
It would be nice if local file path are automatically detected to restore the 1.0.0 behavior.

**Comment by astefanutti** [comment]
Ah indeed. It looks like Puppeteer requires the scheme to be specified: https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagegotourl-options.

It'd be possible to default to the `file` scheme at the level of DeckTape. At first though that sounds like a sensible idea.

**Comment by patrickdepinguin** [comment]
Bumped into the same problem today.
At a minimum, the example in the README could be updated if the actual suggestion takes more work.

**Comment by astefanutti** [comment]
@patrickdepinguin thanks for the feedback. I'm planning on implementing that default ASAP.

**Comment by pat-s** [comment]
This should be fixed or populated in the README - took me two hours to figure out whats going on :/

**Comment by astefanutti** [comment]
@pat-s sorry to hear that. I plan to fix it ASAP.

**Comment by astefanutti** [comment]
Fixed in version `2.5.0`.

**Comment by pat-s** [comment]
Thanks, works :+1:
